src/app/shared/models/book.model.ts

```diff
@@ -0,0 +1,5 @@
+export class Book {
```

Comment:
Pls use `I` prefix for interfaces.
`IBook`.
This prefix helps us to understand what this type exactly is without looking at it.